setup.cfg

```diff
@@ -1,81 +1,108 @@
 [metadata]
 name = virtualenv
+version = attr: virtualenv.__version__
 description = Virtual Python Environment builder
 long_description = file: README.rst
 keywords = virtual, environments, isolated
 maintainer = Bernat Gabor
-author = Ian Bicking
+author = Bernat Gabor
 maintainer-email = gaborjbernat@gmail.com
-author-email = ianb@colorstudy.com
+author-email = gaborjbernat@gmail.com
 url = https://virtualenv.pypa.io/
 project_urls =
     Source=https://github.com/pypa/virtualenv
     Tracker=https://github.com/pypa/virtualenv/issues
-classifiers = Development Status :: 5 - Production/Stable
-              Intended Audience :: Developers
-              License :: OSI Approved :: MIT License
-              Operating System :: POSIX
-              Operating System :: Microsoft :: Windows
-              Operating System :: MacOS :: MacOS X
-              Topic :: Software Development :: Testing
-              Topic :: Software Development :: Libraries
-              Topic :: Utilities
-              Programming Language :: Python :: 2
-              Programming Language :: Python :: 2.7
-              Programming Language :: Python :: 3
-              Programming Language :: Python :: 3.4
-              Programming Language :: Python :: 3.5
-              Programming Language :: Python :: 3.6
-              Programming Language :: Python :: 3.7
+classifiers =
+    Development Status :: 3 - Alpha
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.4
```

Comment:
Probably also minor:

I don't see 3.4 running under CI -- assuming that's intentional (though 3.4 is dead anyhow isn't it?) it's a bit a matter of style as to whether you put it here I guess so maybe it's fine -- just pointing it out to be sure you meant to.

Reply:
https://github.com/pypa/virtualenv/issues/1472